Source/Fuse.PushNotifications/JS.uno

```diff
@@ -41,7 +41,7 @@ namespace Fuse.PushNotifications
 				"registrationSucceeded")
 		{
 			if (_instance != null) return;
-			Resource.SetGlobalKey(_instance = this, "FuseJS/Push");
+			Uno.UX.Resource.SetGlobalKey(_instance = this, "FuseJS/Push");
```

Comment:
without `Uno.UX ` part,  compiler shows `Resource is ambiguous`. If you're certain it can work without any error we can revert this change to the old thing

Reply:
I don't see anything in this commit that introduces this ambiguity, so I don`t think this is the correct commit to do this in. Please correct me if I'm wrong.

Reply:
This is the error that I'm encountering for `Resource`

```
Fuse.PushNotifications/JS.uno(44,5): Error E3109: 'Resource' is ambiguous
Candidates are: Uno.UX.Resource
                Fuse.Reactive.Resource
```

Reply:
Hmm, this sounds familiar. Does this also happen before this patch-series? If so, it probably deserves its own fix, and possibly a back-port...

Reply:
Yes
this happens to me before this patch-series